<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>TabulaMurisSenisData • TabulaMurisSenisData</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="TabulaMurisSenisData">
<meta property="og:description" content="TabulaMurisSenisData">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">TabulaMurisSenisData</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/TabulaMurisSenisData.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/fmicompbio/TabulaMurisSenisData/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>TabulaMurisSenisData</h1>
                        <h4 data-toc-skip class="author">Charlotte Soneson, Dania Machlab, Federico Marini</h4>
            
            <h4 data-toc-skip class="date">27 October 2021</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/fmicompbio/TabulaMurisSenisData/blob/HEAD/vignettes/TabulaMurisSenisData.Rmd" class="external-link"><code>vignettes/TabulaMurisSenisData.Rmd</code></a></small>
      <div class="hidden name"><code>TabulaMurisSenisData.Rmd</code></div>

    </div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">SingleCellExperiment</span><span class="op">)</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/fmicompbio/TabulaMurisSenisData" class="external-link">TabulaMurisSenisData</a></span><span class="op">)</span>
<span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://ggplot2.tidyverse.org" class="external-link">ggplot2</a></span><span class="op">)</span></code></pre></div>
<div class="section level1">
<h1 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h1>
<p>This package provides access to the processed bulk and single-cell RNA-seq data from the <em>Tabula Muris Senis</em> data set <span class="citation">(Schaum et al. 2019; Tabula Muris Consortium 2020)</span>. The processed bulk RNA-seq data was downloaded from GEO (accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132040" class="external-link">GSE132040</a>). The single-cell data (droplet and FACS) was obtained from FigShare (see below for links). All data sets are provided as <code>SingleCellExperiment</code> objects for easy access and use within the Bioconductor ecosystem.</p>
</div>
<div class="section level1">
<h1 id="bulk-rna-seq-data">Bulk RNA-seq data<a class="anchor" aria-label="anchor" href="#bulk-rna-seq-data"></a>
</h1>
<p>The bulk RNA-seq data can be accessed via the <code><a href="../reference/TabulaMurisSenisBulk.html">TabulaMurisSenisBulk()</a></code> function. By setting the argument <code>infoOnly</code> to <code>TRUE</code>, we can get information about the size of the data set before downloading it.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">tmp</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisBulk.html">TabulaMurisSenisBulk</a></span><span class="op">(</span>infoOnly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; Total download size: 59.8 MiB</span>
<span class="va">tms_bulk</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisBulk.html">TabulaMurisSenisBulk</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="va">tms_bulk</span>
<span class="co">#&gt; class: SingleCellExperiment </span>
<span class="co">#&gt; dim: 54352 947 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(54352): 0610005C13Rik 0610006L08Rik ... n-TSaga9 n-TStga1</span>
<span class="co">#&gt; rowData names(8): source type ... havana_gene tag</span>
<span class="co">#&gt; colnames(947): A1_384Bulk_Plate1_S1 A1_384Bulk_Plate3_S1 ...</span>
<span class="co">#&gt;   P9_384Bulk_Plate2_S369 P9_384Bulk_Plate3_S369</span>
<span class="co">#&gt; colData names(19): Sample name title ... __alignment_not_unique organ</span>
<span class="co">#&gt; reducedDimNames(0):</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span></code></pre></div>
<p>We list the available tissues.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">tms_bulk</span><span class="op">)</span><span class="op">$</span><span class="va">organ</span><span class="op">)</span>
<span class="co">#&gt; </span>
<span class="co">#&gt;             BAT            Bone           Brain             GAT           Heart </span>
<span class="co">#&gt;              54              55              56              56              54 </span>
<span class="co">#&gt;          Kidney     Limb_Muscle           Liver            Lung          Marrow </span>
<span class="co">#&gt;              55              54              55              55              54 </span>
<span class="co">#&gt;             MAT              NA        Pancreas            SCAT            Skin </span>
<span class="co">#&gt;              56              14              56              56              51 </span>
<span class="co">#&gt; Small_Intestine          Spleen             WBC </span>
<span class="co">#&gt;              55              56              55</span></code></pre></div>
</div>
<div class="section level1">
<h1 id="single-cell-rna-seq-data">Single-cell RNA-seq data<a class="anchor" aria-label="anchor" href="#single-cell-rna-seq-data"></a>
</h1>
<div class="section level2">
<h2 id="droplet-data">Droplet data<a class="anchor" aria-label="anchor" href="#droplet-data"></a>
</h2>
<p>The data files for the droplet single-cell RNA-seq data were downloaded from FigShare:</p>
<ul>
<li>the <a href="https://figshare.com/articles/dataset/Processed_files_to_use_with_scanpy_/8273102?file=23938934" class="external-link">full data (all tissues)</a>
</li>
<li>the <a href="https://figshare.com/articles/dataset/Tabula_Muris_Senis_Data_Objects/12654728" class="external-link">data from individual tissues</a>
</li>
</ul>
<p>We list the available tissues.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/listTabulaMurisSenisTissues.html">listTabulaMurisSenisTissues</a></span><span class="op">(</span>dataset <span class="op">=</span> <span class="st">"Droplet"</span><span class="op">)</span>
<span class="co">#&gt;  [1] "All"             "Large_Intestine" "Pancreas"        "Trachea"        </span>
<span class="co">#&gt;  [5] "Skin"            "Fat"             "Thymus"          "Liver"          </span>
<span class="co">#&gt;  [9] "Heart_and_Aorta" "Mammary_Gland"   "Bladder"         "Lung"           </span>
<span class="co">#&gt; [13] "Kidney"          "Limb_Muscle"     "Spleen"          "Tongue"         </span>
<span class="co">#&gt; [17] "Marrow"</span></code></pre></div>
<p>As for the bulk data, we can print the size of the data set before downloading it.</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">tmp</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisDroplet.html">TabulaMurisSenisDroplet</a></span><span class="op">(</span>tissues <span class="op">=</span> <span class="st">"All"</span>, infoOnly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; Total download size (All): 709.0 MiB</span>
<span class="va">tms_droplet</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisDroplet.html">TabulaMurisSenisDroplet</a></span><span class="op">(</span>tissues <span class="op">=</span> <span class="st">"All"</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; require("rhdf5")</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="va">tms_droplet</span>
<span class="co">#&gt; $All</span>
<span class="co">#&gt; class: SingleCellExperiment </span>
<span class="co">#&gt; dim: 20138 245389 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(20138): Xkr4 Rp1 ... Sly Erdr1</span>
<span class="co">#&gt; rowData names(6): n_cells means ... highly_variable varm</span>
<span class="co">#&gt; colnames(245389): AAACCTGCAGGGTACA-1-0-0-0 AAACCTGCAGTAAGCG-1-0-0-0 ...</span>
<span class="co">#&gt;   10X_P8_15_TTTGTCATCGGCTTGG-1 10X_P8_15_TTTGTCATCTTGTTTG-1</span>
<span class="co">#&gt; colData names(15): age cell ... louvain leiden</span>
<span class="co">#&gt; reducedDimNames(2): PCA UMAP</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span></code></pre></div>
<p>We plot the UMAP of the entire data set and color by tissue, to re-create the plot from <a href="https://tabula-muris-senis.ds.czbiohub.org" class="external-link">here</a>.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># tissue colors</span>
<span class="va">tissue_cols</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>Pancreas <span class="op">=</span> <span class="st">"#3182bd"</span>, Thymus <span class="op">=</span> <span class="st">"#31a354"</span>, 
                 Trachea <span class="op">=</span> <span class="st">"#636363"</span>, Bladder <span class="op">=</span> <span class="st">"#637939"</span>, 
                 Lung <span class="op">=</span> <span class="st">"#7b4173"</span>, Large_Intestine <span class="op">=</span> <span class="st">"#843c39"</span>, 
                 Fat <span class="op">=</span> <span class="st">"#969696"</span>, Tongue <span class="op">=</span> <span class="st">"#a1d99b"</span>, 
                 Mammary_Gland <span class="op">=</span> <span class="st">"#ce6dbd"</span>, Limb_Muscle <span class="op">=</span> <span class="st">"#d6616b"</span>, 
                 Marrow <span class="op">=</span> <span class="st">"#de9ed6"</span>, Skin <span class="op">=</span> <span class="st">"#e6550d"</span>, 
                 Liver <span class="op">=</span> <span class="st">"#e7969c"</span>, Heart_and_Aorta <span class="op">=</span> <span class="st">"#e7ba52"</span>, 
                 Kidney <span class="op">=</span> <span class="st">"#e7cb94"</span>, Spleen <span class="op">=</span> <span class="st">"#fd8d3c"</span><span class="op">)</span>

<span class="co"># get dataset with all tissues</span>
<span class="va">se</span> <span class="op">&lt;-</span> <span class="va">tms_droplet</span><span class="op">$</span><span class="va">All</span>
<span class="va">se</span> 
<span class="co">#&gt; class: SingleCellExperiment </span>
<span class="co">#&gt; dim: 20138 245389 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(20138): Xkr4 Rp1 ... Sly Erdr1</span>
<span class="co">#&gt; rowData names(6): n_cells means ... highly_variable varm</span>
<span class="co">#&gt; colnames(245389): AAACCTGCAGGGTACA-1-0-0-0 AAACCTGCAGTAAGCG-1-0-0-0 ...</span>
<span class="co">#&gt;   10X_P8_15_TTTGTCATCGGCTTGG-1 10X_P8_15_TTTGTCATCTTGTTTG-1</span>
<span class="co">#&gt; colData names(15): age cell ... louvain leiden</span>
<span class="co">#&gt; reducedDimNames(2): PCA UMAP</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span>

<span class="co"># prepare data set for ggplot</span>
<span class="va">ds</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SingleCellExperiment/man/reducedDims.html" class="external-link">reducedDim</a></span><span class="op">(</span><span class="va">se</span>, <span class="st">"UMAP"</span><span class="op">)</span><span class="op">)</span>
<span class="va">ds</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="va">ds</span>, tissue <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span><span class="op">$</span><span class="va">tissue</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">ds</span><span class="op">)</span>
<span class="co">#&gt;                              UMAP1      UMAP2 tissue</span>
<span class="co">#&gt; AAACCTGCAGGGTACA-1-0-0-0 5.5556602 -10.160711 Tongue</span>
<span class="co">#&gt; AAACCTGCAGTAAGCG-1-0-0-0 2.9584570 -14.145093 Tongue</span>
<span class="co">#&gt; AAACCTGTCATTATCC-1-0-0-0 3.1235533 -14.481063 Tongue</span>
<span class="co">#&gt; AAACGGGGTACAGTGG-1-0-0-0 1.5939721 -14.062417 Tongue</span>
<span class="co">#&gt; AAACGGGGTCTTCTCG-1-0-0-0 0.5233619  -8.997872 Tongue</span>
<span class="co">#&gt; AAAGATGAGCTATGCT-1-0-0-0 1.0210617 -14.642970 Tongue</span>

<span class="co"># plot</span>
<span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">ds</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">UMAP1</span>, y <span class="op">=</span> <span class="va">UMAP2</span>, color <span class="op">=</span> <span class="va">tissue</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">0.05</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_manual.html" class="external-link">scale_color_manual</a></span><span class="op">(</span>values <span class="op">=</span> <span class="va">tissue_cols</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggtheme.html" class="external-link">theme_classic</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/guides.html" class="external-link">guides</a></span><span class="op">(</span>colour <span class="op">=</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/guide_legend.html" class="external-link">guide_legend</a></span><span class="op">(</span>override.aes <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">5</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<p><img src="TabulaMurisSenisData_files/figure-html/unnamed-chunk-6-1.png" width="700"></p>
</div>
<div class="section level2">
<h2 id="facs-data">FACS data<a class="anchor" aria-label="anchor" href="#facs-data"></a>
</h2>
<p>The data files for the FACS single-cell RNA-seq data were downloaded from FigShare:</p>
<ul>
<li>the <a href="https://figshare.com/articles/dataset/Processed_files_to_use_with_scanpy_/8273102?file=23937842" class="external-link">full data (all tissues)</a>
</li>
<li>the <a href="https://figshare.com/articles/dataset/Tabula_Muris_Senis_Data_Objects/12654728" class="external-link">data from individual tissues</a>
</li>
</ul>
<p>We list the available tissues.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/listTabulaMurisSenisTissues.html">listTabulaMurisSenisTissues</a></span><span class="op">(</span>dataset <span class="op">=</span> <span class="st">"FACS"</span><span class="op">)</span>
<span class="co">#&gt;  [1] "All"               "Aorta"             "Kidney"           </span>
<span class="co">#&gt;  [4] "Diaphragm"         "BAT"               "Spleen"           </span>
<span class="co">#&gt;  [7] "Limb_Muscle"       "Liver"             "MAT"              </span>
<span class="co">#&gt; [10] "Thymus"            "Trachea"           "GAT"              </span>
<span class="co">#&gt; [13] "SCAT"              "Bladder"           "Lung"             </span>
<span class="co">#&gt; [16] "Mammary_Gland"     "Pancreas"          "Skin"             </span>
<span class="co">#&gt; [19] "Tongue"            "Brain_Non-Myeloid" "Heart"            </span>
<span class="co">#&gt; [22] "Brain_Myeloid"     "Large_Intestine"   "Marrow"</span></code></pre></div>
<p>Also here, we can print the size of the data set before downloading it.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">tmp</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisFACS.html">TabulaMurisSenisFACS</a></span><span class="op">(</span>tissues <span class="op">=</span> <span class="st">"All"</span>, infoOnly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; Total download size (All): 697.0 MiB</span>
<span class="va">tms_facs</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisFACS.html">TabulaMurisSenisFACS</a></span><span class="op">(</span>tissues <span class="op">=</span> <span class="st">"All"</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="va">tms_facs</span>
<span class="co">#&gt; $All</span>
<span class="co">#&gt; class: SingleCellExperiment </span>
<span class="co">#&gt; dim: 22966 110824 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(22966): 0610005C13Rik 0610007C21Rik ... Zzef1 Zzz3</span>
<span class="co">#&gt; rowData names(6): n_cells means ... highly_variable varm</span>
<span class="co">#&gt; colnames(110824): A10_B000497_B009023_S10.mm10-plus-0-0</span>
<span class="co">#&gt;   A10_B000756_B007446_S10.mm10-plus-0-0 ... P9_B000492_S153.mus-2-1</span>
<span class="co">#&gt;   P9_MAA001700_S105.mus-2-1</span>
<span class="co">#&gt; colData names(15): FACS.selection age ... louvain leiden</span>
<span class="co">#&gt; reducedDimNames(2): PCA UMAP</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span></code></pre></div>
<p>We plot the UMAP of the entire data set and color by tissue, to re-create the plot from <a href="https://tabula-muris-senis.ds.czbiohub.org" class="external-link">here</a>.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="co"># tissue colors</span>
<span class="va">tissue_cols</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>Skin <span class="op">=</span> <span class="st">"#e6550d"</span>, Pancreas <span class="op">=</span> <span class="st">"#3182bd"</span>, 
                 Limb_Muscle <span class="op">=</span> <span class="st">"#d6616b"</span>, Heart <span class="op">=</span> <span class="st">"#e7ba52"</span>, 
                 Spleen <span class="op">=</span> <span class="st">"#fd8d3c"</span>, Diaphragm <span class="op">=</span> <span class="st">"#8c6d31"</span>, 
                 Trachea <span class="op">=</span> <span class="st">"#636363"</span>, Tongue <span class="op">=</span> <span class="st">"#a1d99b"</span>, 
                 Thymus <span class="op">=</span> <span class="st">"#31a354"</span>, `Brain_Non-Myeloid` <span class="op">=</span> <span class="st">"#cedb9c"</span>, 
                 Brain_Myeloid <span class="op">=</span> <span class="st">"#b5cf6b"</span>, Bladder <span class="op">=</span> <span class="st">"#637939"</span>, 
                 Large_Intestine <span class="op">=</span> <span class="st">"#843c39"</span>, BAT <span class="op">=</span> <span class="st">"#9c9ede"</span>, 
                 GAT <span class="op">=</span> <span class="st">"#bd9e39"</span>, MAT <span class="op">=</span> <span class="st">"#a55194"</span>, SCAT <span class="op">=</span> <span class="st">"#6baed6"</span>, 
                 Lung <span class="op">=</span> <span class="st">"#7b4173"</span>, Liver <span class="op">=</span> <span class="st">"#e7969c"</span>, 
                 Marrow <span class="op">=</span> <span class="st">"#de9ed6"</span>, Kidney <span class="op">=</span> <span class="st">"#e7cb94"</span>, 
                 Aorta <span class="op">=</span> <span class="st">"#393b79"</span>, Mammary_Gland <span class="op">=</span> <span class="st">"#ce6dbd"</span><span class="op">)</span>

<span class="co"># get dataset with all tissues</span>
<span class="va">se</span> <span class="op">&lt;-</span> <span class="va">tms_facs</span><span class="op">$</span><span class="va">All</span>
<span class="va">se</span> 
<span class="co">#&gt; class: SingleCellExperiment </span>
<span class="co">#&gt; dim: 22966 110824 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(1): counts</span>
<span class="co">#&gt; rownames(22966): 0610005C13Rik 0610007C21Rik ... Zzef1 Zzz3</span>
<span class="co">#&gt; rowData names(6): n_cells means ... highly_variable varm</span>
<span class="co">#&gt; colnames(110824): A10_B000497_B009023_S10.mm10-plus-0-0</span>
<span class="co">#&gt;   A10_B000756_B007446_S10.mm10-plus-0-0 ... P9_B000492_S153.mus-2-1</span>
<span class="co">#&gt;   P9_MAA001700_S105.mus-2-1</span>
<span class="co">#&gt; colData names(15): FACS.selection age ... louvain leiden</span>
<span class="co">#&gt; reducedDimNames(2): PCA UMAP</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span>

<span class="co"># prepare data set for ggplot</span>
<span class="va">ds</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/as.data.frame.html" class="external-link">as.data.frame</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/pkg/SingleCellExperiment/man/reducedDims.html" class="external-link">reducedDim</a></span><span class="op">(</span><span class="va">se</span>, <span class="st">"UMAP"</span><span class="op">)</span><span class="op">)</span>
<span class="va">ds</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/cbind.html" class="external-link">cbind</a></span><span class="op">(</span><span class="va">ds</span>, tissue <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">colData</a></span><span class="op">(</span><span class="va">se</span><span class="op">)</span><span class="op">$</span><span class="va">tissue</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">ds</span><span class="op">)</span>
<span class="co">#&gt;                                           UMAP1      UMAP2      tissue</span>
<span class="co">#&gt; A10_B000497_B009023_S10.mm10-plus-0-0  9.917321  10.959189        Skin</span>
<span class="co">#&gt; A10_B000756_B007446_S10.mm10-plus-0-0 12.558255 -14.239025    Pancreas</span>
<span class="co">#&gt; A10_B000802_B009022_S10.mm10-plus-0-0 10.272095   9.065876        Skin</span>
<span class="co">#&gt; A10_B000927_B007456_S10.mm10-plus-0-0  1.349453   7.980347 Limb_Muscle</span>
<span class="co">#&gt; A10_B001361_B007505_S10.mm10-plus-0-0  4.771871  -7.167833       Heart</span>
<span class="co">#&gt; A10_B002452_B009020_S10.mm10-plus-0-0  3.218485  -8.522328      Spleen</span>

<span class="co"># plot</span>
<span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggplot.html" class="external-link">ggplot</a></span><span class="op">(</span><span class="va">ds</span>, <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/aes.html" class="external-link">aes</a></span><span class="op">(</span>x <span class="op">=</span> <span class="va">UMAP1</span>, y <span class="op">=</span> <span class="va">UMAP2</span>, color <span class="op">=</span> <span class="va">tissue</span><span class="op">)</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/geom_point.html" class="external-link">geom_point</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">0.05</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/scale_manual.html" class="external-link">scale_color_manual</a></span><span class="op">(</span>values <span class="op">=</span> <span class="va">tissue_cols</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/ggtheme.html" class="external-link">theme_classic</a></span><span class="op">(</span><span class="op">)</span> <span class="op">+</span> 
  <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/guides.html" class="external-link">guides</a></span><span class="op">(</span>colour <span class="op">=</span> <span class="fu"><a href="https://ggplot2.tidyverse.org/reference/guide_legend.html" class="external-link">guide_legend</a></span><span class="op">(</span>override.aes <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>size <span class="op">=</span> <span class="fl">5</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></code></pre></div>
<p><img src="TabulaMurisSenisData_files/figure-html/unnamed-chunk-9-1.png" width="700"></p>
</div>
</div>
<div class="section level1">
<h1 id="explore-data-with-isee">Explore data with <code>iSEE</code><a class="anchor" aria-label="anchor" href="#explore-data-with-isee"></a>
</h1>
<p>The <em>Tabula Muris Senis</em> datasets are provided in the form of <code>SingleCellExperiment</code> objects.<br>
A natural companion to this data structure is the <em><a href="https://bioconductor.org/packages/3.14/iSEE" class="external-link">iSEE</a></em> package, which can be used for interactive and reproducible data exploration.</p>
<p>Any analysis steps should be performed in advance before calling <code>iSEE</code>, and since these datasets can be quite big, the operations can be time consuming, and/or require a considerable amount of resources.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">sce_all_facs</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TabulaMurisSenisFACS.html">TabulaMurisSenisFACS</a></span><span class="op">(</span>tissues <span class="op">=</span> <span class="st">"All"</span>, processedCounts <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">$</span><span class="va">All</span>
<span class="co">#&gt; snapshotDate(): 2021-10-18</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="co">#&gt; see ?TabulaMurisSenisData and browseVignettes('TabulaMurisSenisData') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="va">sce_all_facs</span>
<span class="co">#&gt; class: SingleCellExperiment </span>
<span class="co">#&gt; dim: 22966 110824 </span>
<span class="co">#&gt; metadata(0):</span>
<span class="co">#&gt; assays(2): counts logcounts</span>
<span class="co">#&gt; rownames(22966): 0610005C13Rik 0610007C21Rik ... Zzef1 Zzz3</span>
<span class="co">#&gt; rowData names(6): n_cells means ... highly_variable varm</span>
<span class="co">#&gt; colnames(110824): A10_B000497_B009023_S10.mm10-plus-0-0</span>
<span class="co">#&gt;   A10_B000756_B007446_S10.mm10-plus-0-0 ... P9_B000492_S153.mus-2-1</span>
<span class="co">#&gt;   P9_MAA001700_S105.mus-2-1</span>
<span class="co">#&gt; colData names(15): FACS.selection age ... louvain leiden</span>
<span class="co">#&gt; reducedDimNames(2): PCA UMAP</span>
<span class="co">#&gt; mainExpName: NULL</span>
<span class="co">#&gt; altExpNames(0):</span></code></pre></div>
<p>We can see that the <code>sce_all_facs</code> contains both raw and processed counts (in the assays slots <code>counts</code> and <code>logcounts</code>), but also the PCA and UMAP embeddings as provided in the original publication.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/pkg/SummarizedExperiment/man/SummarizedExperiment-class.html" class="external-link">assayNames</a></span><span class="op">(</span><span class="va">sce_all_facs</span><span class="op">)</span>
<span class="co">#&gt; [1] "counts"    "logcounts"</span>
<span class="fu"><a href="https://rdrr.io/pkg/SingleCellExperiment/man/reducedDims.html" class="external-link">reducedDimNames</a></span><span class="op">(</span><span class="va">sce_all_facs</span><span class="op">)</span>
<span class="co">#&gt; [1] "PCA"  "UMAP"</span></code></pre></div>
<p>If desired, additional processing steps can be performed on the <code>sce_all_facs</code> - e.g., storing signature scores (computed via <em><a href="https://bioconductor.org/packages/3.14/AUCell" class="external-link">AUCell</a></em>) as <code>colData</code> elements.<br>
Once these steps are completed, launching <code>iSEE</code> is as easy as</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/iSEE/iSEE" class="external-link">iSEE</a></span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE</a></span><span class="op">(</span><span class="va">sce_all_facs</span><span class="op">)</span></code></pre></div>
<p>This will launch <code>iSEE</code> with a standard default set of panels.<br>
Optionally, we can configure the <code>initial</code> set to start the app in the desired configuration - below, we show how to start <code>iSEE</code> with:</p>
<ul>
<li>a <code>ReducedDimensionPlot</code> displaying the UMAP, colored by the expression of a feature;</li>
<li>a <code>RowDataTable</code>, for selecting a feature to be transmitted to the other panels (showing all genes whose name contains “Col1”);</li>
<li>a <code>FeatureAssayPlot</code>, for showing the expression of the gene selected in the <code>RowDataTable</code>, split by tissue.</li>
</ul>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">initial</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span>

<span class="co">################################################################################</span>
<span class="co"># (Compact) Settings for Reduced dimension plot 1</span>
<span class="co">################################################################################</span>

<span class="va">initial</span><span class="op">[[</span><span class="st">"ReducedDimensionPlot1"</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu">new</span><span class="op">(</span>
    <span class="st">"ReducedDimensionPlot"</span>, 
    DataBoxOpen <span class="op">=</span> <span class="cn">TRUE</span>, 
    Type <span class="op">=</span> <span class="st">"UMAP"</span>, 
    VisualBoxOpen <span class="op">=</span> <span class="cn">TRUE</span>, 
    ColorBy <span class="op">=</span> <span class="st">"Feature name"</span>, 
    ColorByFeatureName <span class="op">=</span> <span class="st">"Col1a1"</span>, 
    ColorByFeatureSource <span class="op">=</span> <span class="st">"RowDataTable1"</span>, 
    ColorByFeatureDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>
<span class="op">)</span>

<span class="co">################################################################################</span>
<span class="co"># (Compact) Settings for Row data table 1</span>
<span class="co">################################################################################</span>

<span class="va">initial</span><span class="op">[[</span><span class="st">"RowDataTable1"</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu">new</span><span class="op">(</span>
    <span class="st">"RowDataTable"</span>, 
    Selected <span class="op">=</span> <span class="st">"Col1a1"</span>, 
    Search <span class="op">=</span> <span class="st">"Col1"</span>
<span class="op">)</span>
    
<span class="co">################################################################################</span>
<span class="co"># (Compact) Settings for Feature assay plot 1</span>
<span class="co">################################################################################</span>

<span class="va">initial</span><span class="op">[[</span><span class="st">"FeatureAssayPlot1"</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu">new</span><span class="op">(</span>
    <span class="st">"FeatureAssayPlot"</span>, 
    DataBoxOpen <span class="op">=</span> <span class="cn">TRUE</span>, 
    Assay <span class="op">=</span> <span class="st">"logcounts"</span>, 
    XAxis <span class="op">=</span> <span class="st">"Column data"</span>, 
    XAxisColumnData <span class="op">=</span> <span class="st">"tissue"</span>, 
    YAxisFeatureName <span class="op">=</span> <span class="st">"Col1a1"</span>, 
    YAxisFeatureSource <span class="op">=</span> <span class="st">"RowDataTable1"</span>
<span class="op">)</span></code></pre></div>
<details><br><summary>
Click here to display the complete configuration chunk<br><br></summary><div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">initial</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span>

<span class="co">################################################################################</span>
<span class="co"># Settings for Reduced dimension plot 1</span>
<span class="co">################################################################################</span>

<span class="va">initial</span><span class="op">[[</span><span class="st">"ReducedDimensionPlot1"</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu">new</span><span class="op">(</span><span class="st">"ReducedDimensionPlot"</span>, Type <span class="op">=</span> <span class="st">"UMAP"</span>, XAxis <span class="op">=</span> <span class="fl">1L</span>, YAxis <span class="op">=</span> <span class="fl">2L</span>, 
    FacetRowByColData <span class="op">=</span> <span class="st">"FACS.selection"</span>, FacetColumnByColData <span class="op">=</span> <span class="st">"FACS.selection"</span>, 
    ColorByColumnData <span class="op">=</span> <span class="st">"FACS.selection"</span>, ColorByFeatureNameAssay <span class="op">=</span> <span class="st">"logcounts"</span>, 
    ColorBySampleNameColor <span class="op">=</span> <span class="st">"#FF0000"</span>, ShapeByColumnData <span class="op">=</span> <span class="st">"FACS.selection"</span>, 
    SizeByColumnData <span class="op">=</span> <span class="st">"n_genes"</span>, FacetRowBy <span class="op">=</span> <span class="st">"None"</span>, FacetColumnBy <span class="op">=</span> <span class="st">"None"</span>, 
    ColorBy <span class="op">=</span> <span class="st">"Feature name"</span>, ColorByDefaultColor <span class="op">=</span> <span class="st">"#000000"</span>, 
    ColorByFeatureName <span class="op">=</span> <span class="st">"Col1a1"</span>, ColorByFeatureSource <span class="op">=</span> <span class="st">"RowDataTable1"</span>, 
    ColorByFeatureDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, ColorBySampleName <span class="op">=</span> <span class="st">"A10_B000497_B009023_S10.mm10-plus-0-0"</span>, 
    ColorBySampleSource <span class="op">=</span> <span class="st">"---"</span>, ColorBySampleDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    ShapeBy <span class="op">=</span> <span class="st">"None"</span>, SizeBy <span class="op">=</span> <span class="st">"None"</span>, SelectionAlpha <span class="op">=</span> <span class="fl">0.1</span>, 
    ZoomData <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/numeric.html" class="external-link">numeric</a></span><span class="op">(</span><span class="fl">0</span><span class="op">)</span>, BrushData <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span>, VisualBoxOpen <span class="op">=</span> <span class="cn">TRUE</span>, 
    VisualChoices <span class="op">=</span> <span class="st">"Color"</span>, ContourAdd <span class="op">=</span> <span class="cn">FALSE</span>, ContourColor <span class="op">=</span> <span class="st">"#0000FF"</span>, 
    PointSize <span class="op">=</span> <span class="fl">1</span>, PointAlpha <span class="op">=</span> <span class="fl">1</span>, Downsample <span class="op">=</span> <span class="cn">FALSE</span>, DownsampleResolution <span class="op">=</span> <span class="fl">200</span>, 
    CustomLabels <span class="op">=</span> <span class="cn">FALSE</span>, CustomLabelsText <span class="op">=</span> <span class="st">"A10_B000497_B009023_S10.mm10-plus-0-0"</span>, 
    FontSize <span class="op">=</span> <span class="fl">1</span>, LegendPointSize <span class="op">=</span> <span class="fl">1</span>, LegendPosition <span class="op">=</span> <span class="st">"Bottom"</span>, 
    HoverInfo <span class="op">=</span> <span class="cn">TRUE</span>, LabelCenters <span class="op">=</span> <span class="cn">FALSE</span>, LabelCentersBy <span class="op">=</span> <span class="st">"FACS.selection"</span>, 
    LabelCentersColor <span class="op">=</span> <span class="st">"#000000"</span>, VersionInfo <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>iSEE <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/structure.html" class="external-link">structure</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>
        <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">2L</span>, <span class="fl">5L</span>, <span class="fl">1L</span><span class="op">)</span><span class="op">)</span>, class <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"package_version"</span>, <span class="st">"numeric_version"</span>
    <span class="op">)</span><span class="op">)</span><span class="op">)</span>, PanelId <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>ReducedDimensionPlot <span class="op">=</span> <span class="fl">1L</span><span class="op">)</span>, PanelHeight <span class="op">=</span> <span class="fl">500L</span>, 
    PanelWidth <span class="op">=</span> <span class="fl">4L</span>, SelectionBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionSource <span class="op">=</span> <span class="st">"---"</span>, 
    ColumnSelectionSource <span class="op">=</span> <span class="st">"---"</span>, DataBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColumnSelectionDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionRestrict <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColumnSelectionRestrict <span class="op">=</span> <span class="cn">FALSE</span>, SelectionHistory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span>

<span class="co">################################################################################</span>
<span class="co"># Settings for Row data table 1</span>
<span class="co">################################################################################</span>

<span class="va">initial</span><span class="op">[[</span><span class="st">"RowDataTable1"</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu">new</span><span class="op">(</span><span class="st">"RowDataTable"</span>, Selected <span class="op">=</span> <span class="st">"Col1a1"</span>, Search <span class="op">=</span> <span class="st">"Col1"</span>, SearchColumns <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">""</span>, 
<span class="st">""</span>, <span class="st">""</span>, <span class="st">""</span>, <span class="st">""</span><span class="op">)</span>, HiddenColumns <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/character.html" class="external-link">character</a></span><span class="op">(</span><span class="fl">0</span><span class="op">)</span>, VersionInfo <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>
    iSEE <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/structure.html" class="external-link">structure</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">2L</span>, <span class="fl">5L</span>, <span class="fl">1L</span><span class="op">)</span><span class="op">)</span>, class <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"package_version"</span>, 
    <span class="st">"numeric_version"</span><span class="op">)</span><span class="op">)</span><span class="op">)</span>, PanelId <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>RowDataTable <span class="op">=</span> <span class="fl">1L</span><span class="op">)</span>, PanelHeight <span class="op">=</span> <span class="fl">500L</span>, 
    PanelWidth <span class="op">=</span> <span class="fl">4L</span>, SelectionBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionSource <span class="op">=</span> <span class="st">"---"</span>, 
    ColumnSelectionSource <span class="op">=</span> <span class="st">"---"</span>, DataBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColumnSelectionDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionRestrict <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColumnSelectionRestrict <span class="op">=</span> <span class="cn">FALSE</span>, SelectionHistory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span>

<span class="co">################################################################################</span>
<span class="co"># Settings for Feature assay plot 1</span>
<span class="co">################################################################################</span>

<span class="va">initial</span><span class="op">[[</span><span class="st">"FeatureAssayPlot1"</span><span class="op">]</span><span class="op">]</span> <span class="op">&lt;-</span> <span class="fu">new</span><span class="op">(</span><span class="st">"FeatureAssayPlot"</span>, Assay <span class="op">=</span> <span class="st">"logcounts"</span>, XAxis <span class="op">=</span> <span class="st">"Column data"</span>, 
    XAxisColumnData <span class="op">=</span> <span class="st">"tissue"</span>, XAxisFeatureName <span class="op">=</span> <span class="st">"0610005C13Rik"</span>, 
    XAxisFeatureSource <span class="op">=</span> <span class="st">"---"</span>, XAxisFeatureDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    YAxisFeatureName <span class="op">=</span> <span class="st">"Col1a1"</span>, YAxisFeatureSource <span class="op">=</span> <span class="st">"RowDataTable1"</span>, 
    YAxisFeatureDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, FacetRowByColData <span class="op">=</span> <span class="st">"FACS.selection"</span>, 
    FacetColumnByColData <span class="op">=</span> <span class="st">"FACS.selection"</span>, ColorByColumnData <span class="op">=</span> <span class="st">"age"</span>, 
    ColorByFeatureNameAssay <span class="op">=</span> <span class="st">"logcounts"</span>, ColorBySampleNameColor <span class="op">=</span> <span class="st">"#FF0000"</span>, 
    ShapeByColumnData <span class="op">=</span> <span class="st">"FACS.selection"</span>, SizeByColumnData <span class="op">=</span> <span class="st">"n_genes"</span>, 
    FacetRowBy <span class="op">=</span> <span class="st">"None"</span>, FacetColumnBy <span class="op">=</span> <span class="st">"None"</span>, ColorBy <span class="op">=</span> <span class="st">"None"</span>, 
    ColorByDefaultColor <span class="op">=</span> <span class="st">"#000000"</span>, ColorByFeatureName <span class="op">=</span> <span class="st">"0610005C13Rik"</span>, 
    ColorByFeatureSource <span class="op">=</span> <span class="st">"---"</span>, ColorByFeatureDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColorBySampleName <span class="op">=</span> <span class="st">"A10_B000497_B009023_S10.mm10-plus-0-0"</span>, 
    ColorBySampleSource <span class="op">=</span> <span class="st">"---"</span>, ColorBySampleDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    ShapeBy <span class="op">=</span> <span class="st">"None"</span>, SizeBy <span class="op">=</span> <span class="st">"None"</span>, SelectionAlpha <span class="op">=</span> <span class="fl">0.1</span>, 
    ZoomData <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/numeric.html" class="external-link">numeric</a></span><span class="op">(</span><span class="fl">0</span><span class="op">)</span>, BrushData <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span>, VisualBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, 
    VisualChoices <span class="op">=</span> <span class="st">"Color"</span>, ContourAdd <span class="op">=</span> <span class="cn">FALSE</span>, ContourColor <span class="op">=</span> <span class="st">"#0000FF"</span>, 
    PointSize <span class="op">=</span> <span class="fl">1</span>, PointAlpha <span class="op">=</span> <span class="fl">1</span>, Downsample <span class="op">=</span> <span class="cn">FALSE</span>, DownsampleResolution <span class="op">=</span> <span class="fl">200</span>, 
    CustomLabels <span class="op">=</span> <span class="cn">FALSE</span>, CustomLabelsText <span class="op">=</span> <span class="st">"A10_B000497_B009023_S10.mm10-plus-0-0"</span>, 
    FontSize <span class="op">=</span> <span class="fl">1</span>, LegendPointSize <span class="op">=</span> <span class="fl">1</span>, LegendPosition <span class="op">=</span> <span class="st">"Bottom"</span>, 
    HoverInfo <span class="op">=</span> <span class="cn">TRUE</span>, LabelCenters <span class="op">=</span> <span class="cn">FALSE</span>, LabelCentersBy <span class="op">=</span> <span class="st">"FACS.selection"</span>, 
    LabelCentersColor <span class="op">=</span> <span class="st">"#000000"</span>, VersionInfo <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>iSEE <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/structure.html" class="external-link">structure</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span>
        <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">2L</span>, <span class="fl">5L</span>, <span class="fl">1L</span><span class="op">)</span><span class="op">)</span>, class <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"package_version"</span>, <span class="st">"numeric_version"</span>
    <span class="op">)</span><span class="op">)</span><span class="op">)</span>, PanelId <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span>FeatureAssayPlot <span class="op">=</span> <span class="fl">1L</span><span class="op">)</span>, PanelHeight <span class="op">=</span> <span class="fl">500L</span>, 
    PanelWidth <span class="op">=</span> <span class="fl">4L</span>, SelectionBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionSource <span class="op">=</span> <span class="st">"---"</span>, 
    ColumnSelectionSource <span class="op">=</span> <span class="st">"---"</span>, DataBoxOpen <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColumnSelectionDynamicSource <span class="op">=</span> <span class="cn">FALSE</span>, RowSelectionRestrict <span class="op">=</span> <span class="cn">FALSE</span>, 
    ColumnSelectionRestrict <span class="op">=</span> <span class="cn">FALSE</span>, SelectionHistory <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/list.html" class="external-link">list</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span>

<span class="va">initial</span></code></pre></div>
<p><br></p>
</details><p><code>iSEE</code> can then be launched with the following command:</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/pkg/iSEE/man/iSEE.html" class="external-link">iSEE</a></span><span class="op">(</span><span class="va">sce_all_facs</span>, initial <span class="op">=</span> <span class="va">initial</span><span class="op">)</span></code></pre></div>
<div class="figure">
<img src="ss_iSEE_facsdataset.jpg" alt="Screenshot of the iSEE app running on the FACS single cell dataset" width="100%"><p class="caption">
Screenshot of the iSEE app running on the FACS single cell dataset
</p>
</div>
<p>Note that these lengthy configuration options can be readily exported by clicking on the dedicated button “Display panel settings” in <code>iSEE</code> - we can do that anytime after we are done designing the configuration using the app interface.</p>
</div>
<div class="section level1 unnumbered">
<h1 class="unnumbered" id="session-info">Session info<a class="anchor" aria-label="anchor" href="#session-info"></a>
</h1>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html" class="external-link">sessionInfo</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; R version 4.1.1 (2021-08-10)</span>
<span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span>
<span class="co">#&gt; Running under: Ubuntu 20.04.3 LTS</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Matrix products: default</span>
<span class="co">#&gt; BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; locale:</span>
<span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span>
<span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span>
<span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             </span>
<span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span>
<span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span>
<span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; attached base packages:</span>
<span class="co">#&gt; [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span>
<span class="co">#&gt; [8] base     </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; other attached packages:</span>
<span class="co">#&gt;  [1] iSEE_2.5.3                  rhdf5_2.37.4               </span>
<span class="co">#&gt;  [3] ggplot2_3.3.5               TabulaMurisSenisData_1.1.0 </span>
<span class="co">#&gt;  [5] SingleCellExperiment_1.15.2 SummarizedExperiment_1.23.5</span>
<span class="co">#&gt;  [7] Biobase_2.53.0              GenomicRanges_1.45.0       </span>
<span class="co">#&gt;  [9] GenomeInfoDb_1.29.10        IRanges_2.27.2             </span>
<span class="co">#&gt; [11] S4Vectors_0.31.5            BiocGenerics_0.39.2        </span>
<span class="co">#&gt; [13] MatrixGenerics_1.5.4        matrixStats_0.61.0         </span>
<span class="co">#&gt; [15] BiocStyle_2.21.4           </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; loaded via a namespace (and not attached):</span>
<span class="co">#&gt;   [1] circlize_0.4.13               AnnotationHub_3.1.7          </span>
<span class="co">#&gt;   [3] BiocFileCache_2.1.1           systemfonts_1.0.3            </span>
<span class="co">#&gt;   [5] igraph_1.2.7                  shinydashboard_0.7.2         </span>
<span class="co">#&gt;   [7] splines_4.1.1                 digest_0.6.28                </span>
<span class="co">#&gt;   [9] foreach_1.5.1                 htmltools_0.5.2              </span>
<span class="co">#&gt;  [11] gdata_2.18.0                  fansi_0.5.0                  </span>
<span class="co">#&gt;  [13] magrittr_2.0.1                memoise_2.0.0                </span>
<span class="co">#&gt;  [15] cluster_2.1.2                 doParallel_1.0.16            </span>
<span class="co">#&gt;  [17] ComplexHeatmap_2.9.4          Biostrings_2.61.2            </span>
<span class="co">#&gt;  [19] pkgdown_1.9000.9000.9000      colorspace_2.0-2             </span>
<span class="co">#&gt;  [21] blob_1.2.2                    rappdirs_0.3.3               </span>
<span class="co">#&gt;  [23] ggrepel_0.9.1                 textshaping_0.3.6            </span>
<span class="co">#&gt;  [25] xfun_0.27                     dplyr_1.0.7                  </span>
<span class="co">#&gt;  [27] crayon_1.4.1                  RCurl_1.98-1.5               </span>
<span class="co">#&gt;  [29] jsonlite_1.7.2                iterators_1.0.13             </span>
<span class="co">#&gt;  [31] glue_1.4.2                    gtable_0.3.0                 </span>
<span class="co">#&gt;  [33] zlibbioc_1.39.0               XVector_0.33.0               </span>
<span class="co">#&gt;  [35] GetoptLong_1.0.5              DelayedArray_0.19.4          </span>
<span class="co">#&gt;  [37] Rhdf5lib_1.15.2               shape_1.4.6                  </span>
<span class="co">#&gt;  [39] HDF5Array_1.21.0              scales_1.1.1                 </span>
<span class="co">#&gt;  [41] DBI_1.1.1                     miniUI_0.1.1.1               </span>
<span class="co">#&gt;  [43] Rcpp_1.0.7                    viridisLite_0.4.0            </span>
<span class="co">#&gt;  [45] xtable_1.8-4                  clue_0.3-60                  </span>
<span class="co">#&gt;  [47] bit_4.0.4                     DT_0.19                      </span>
<span class="co">#&gt;  [49] htmlwidgets_1.5.4             httr_1.4.2                   </span>
<span class="co">#&gt;  [51] RColorBrewer_1.1-2            shinyAce_0.4.1               </span>
<span class="co">#&gt;  [53] ellipsis_0.3.2                pkgconfig_2.0.3              </span>
<span class="co">#&gt;  [55] farver_2.1.0                  sass_0.4.0                   </span>
<span class="co">#&gt;  [57] dbplyr_2.1.1                  utf8_1.2.2                   </span>
<span class="co">#&gt;  [59] tidyselect_1.1.1              labeling_0.4.2               </span>
<span class="co">#&gt;  [61] rlang_0.99.0.9000             later_1.3.0                  </span>
<span class="co">#&gt;  [63] AnnotationDbi_1.55.2          munsell_0.5.0                </span>
<span class="co">#&gt;  [65] BiocVersion_3.14.0            tools_4.1.1                  </span>
<span class="co">#&gt;  [67] cachem_1.0.6                  cli_3.0.1                    </span>
<span class="co">#&gt;  [69] generics_0.1.1                RSQLite_2.2.8                </span>
<span class="co">#&gt;  [71] ExperimentHub_2.1.4           rintrojs_0.3.0               </span>
<span class="co">#&gt;  [73] evaluate_0.14                 stringr_1.4.0                </span>
<span class="co">#&gt;  [75] fastmap_1.1.0                 yaml_2.2.1                   </span>
<span class="co">#&gt;  [77] ragg_1.1.3                    knitr_1.36                   </span>
<span class="co">#&gt;  [79] bit64_4.0.5                   fs_1.5.0                     </span>
<span class="co">#&gt;  [81] purrr_0.3.4                   KEGGREST_1.33.0              </span>
<span class="co">#&gt;  [83] nlme_3.1-153                  mime_0.12                    </span>
<span class="co">#&gt;  [85] compiler_4.1.1                filelock_1.0.2               </span>
<span class="co">#&gt;  [87] curl_4.3.2                    png_0.1-7                    </span>
<span class="co">#&gt;  [89] interactiveDisplayBase_1.31.2 tibble_3.1.5                 </span>
<span class="co">#&gt;  [91] bslib_0.3.1                   stringi_1.7.5                </span>
<span class="co">#&gt;  [93] highr_0.9                     desc_1.4.0                   </span>
<span class="co">#&gt;  [95] lattice_0.20-45               Matrix_1.3-4                 </span>
<span class="co">#&gt;  [97] shinyjs_2.0.0                 vctrs_0.3.8                  </span>
<span class="co">#&gt;  [99] pillar_1.6.4                  lifecycle_1.0.1              </span>
<span class="co">#&gt; [101] rhdf5filters_1.5.0            BiocManager_1.30.16          </span>
<span class="co">#&gt; [103] jquerylib_0.1.4               GlobalOptions_0.1.2          </span>
<span class="co">#&gt; [105] bitops_1.0-7                  httpuv_1.6.3                 </span>
<span class="co">#&gt; [107] R6_2.5.1                      bookdown_0.24                </span>
<span class="co">#&gt; [109] promises_1.2.0.1              vipor_0.4.5                  </span>
<span class="co">#&gt; [111] codetools_0.2-18              colourpicker_1.1.1           </span>
<span class="co">#&gt; [113] gtools_3.9.2                  assertthat_0.2.1             </span>
<span class="co">#&gt; [115] rprojroot_2.0.2               rjson_0.2.20                 </span>
<span class="co">#&gt; [117] shinyWidgets_0.6.2            withr_2.4.2                  </span>
<span class="co">#&gt; [119] GenomeInfoDbData_1.2.7        mgcv_1.8-38                  </span>
<span class="co">#&gt; [121] parallel_4.1.1                grid_4.1.1                   </span>
<span class="co">#&gt; [123] rmarkdown_2.11                shiny_1.7.1</span></code></pre></div>
</div>
<div class="section level1 unnumbered">
<h1 class="unnumbered" id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h1>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Schaum2019-nf" class="csl-entry">
Schaum, Nicholas, Benoit Lehallier, Oliver Hahn, Shayan Hosseinzadeh, Song E Lee, Rene Sit, Davis P Lee, et al. 2019. <span>“The Murine Transcriptome Reveals Global Aging Nodes with Organ-Specific Phase and Amplitude.”</span>
</div>
<div id="ref-Tabula_Muris_Consortium2020-um" class="csl-entry">
Tabula Muris Consortium. 2020. <span>“A Single-Cell Transcriptomic Atlas Characterizes Ageing Tissues in the Mouse.”</span> <em>Nature</em> 583 (7817): 590–95.
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Charlotte Soneson, Dania Machlab, Federico Marini, Stefania Astrologo.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 1.9000.9000.9000.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
